### ISSUE TITLE
It fails to open scanner for Samsung Device

### ISSUE THREAD

**Issue by swap9391** [myinnos/Google-Code-Scanner#1]
Device - Samsung S20+ 

com.google.mlkit.common.MlKitException: Failed to scan code.

**Comment by dineshrajamanikam** [comment]
Yes, I'm also facing this same issue. did you find any solution for this issue?

**Comment by myinnos** [comment]
@swap9391 @dineshrajamanikam 
it will be great if I get more error logs to check the exact issue.

**Comment by bhavikpatel** [comment]
I also face not able to open scan in Opo and kopa mobile
